RSAKeyVaultProvider/RSAKeyVault.cs

```diff
@@ -134,5 +133,37 @@ public override byte[] EncryptValue(byte[] rgb)
         {
             throw new NotSupportedException();
         }
+#endif
+
+        private static HashAlgorithm GetHashAlgorithm(HashAlgorithmName algorithm)
+        {
+#if NETSTANDARD20
```

Comment:
Thanks for taking a look. 

I like `CryptoConfig` because it allows registering and resolving algorithm instances without having to recompile intermediate libs. That said, if you believe it's better to have a single code path, I won't fight for it (but you'll have to convince @onovotny we shouldn't have two TFMs). 

For SHA1, sounds like a good idea 👍